implementation/parallel-to-sns-scatter-gather/scatter_gather/lambda_/aggregator/app.py

```diff
@@ -22,26 +22,36 @@
 import json
 import logging
 import boto3
-
+import os
 logging.basicConfig(format='%(asctime)s %(levelname)-8s %(message)s', level=logging.INFO, datefmt='%Y-%m-%d %H:%M:%S')
 logging.getLogger().setLevel(logging.INFO)
 
-sqs = boto3.client('sqs')
+QUOTE_TABLE_NAME = os.environ['QUOTE_TABLE_NAME']
+ddb_table = boto3.resource('dynamodb').Table(QUOTE_TABLE_NAME)
+
 
+# The lambda function receives the message from the SQS event aggregates them and stores it in the DDB table
```

Comment:
IS there a way to get the results from the table? E.g,. a Step Functions Task that reads from DDB?

Reply:
The solution does not provide that feature(I considered it out of scope).  For both before and after refactoring  the solutions stop at aggregation and the results are not consumed/retrieved. Is that ok?